runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -397,6 +399,14 @@ void expand(PluginPanel panel)
 		{
 			navContainer.remove(0);
 		}
+		else
+		{
+			clientSize = this.getSize();
+			if (isInScreenBounds((int) getLocationOnScreen().getX() + getWidth() + PANEL_EXPANDED_WIDTH, (int) getLocationOnScreen().getY()))
```

Comment:
Is there a specific reason for using `(int) getLocationOnScreen().getX()` rather than `getLocationOnScreen().x`? `java.awt.Point`s seem to have an integer `x` field